pathFollower:
  ros__parameters:
    # sensorOffsetX: 0.0  #公共参数，这里修改注意修改其他地方
    # sensorOffsetY: 0.0  #公共参数，这里修改注意修改其他地方
    pubSkipNum: 1
    # twoWayDrive: true #公共参数，这里修改注意修改其他地方
    lookAheadDis: 0.5
    yawRateGain: 7.5
    stopYawRateGain: 7.5
    maxYawRate: 90.0
    # maxSpeed: 2.0
    maxAccel: 2.5
    switchTimeThre: 1.0
    dirDiffThre: 0.1
    stopDisThre: 0.2
    slowDwnDisThre: 0.85
    useInclRateToSlow: false
    inclRateThre: 120.0
    slowRate1: 0.25
    slowRate2: 0.5
    slowTime1: 2.0
    slowTime2: 2.0
    useInclToStop: false
    inclThre: 45.0
    stopTime: 5.0
    noRotAtStop: false
    noRotAtGoal: true
    # autonomyMode: true
    # autonomySpeed: 2.0
    # joyToSpeedDelay: 2.0
    test_param: true